components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -255,11 +255,17 @@ define([
                 var label = new Element('label', {'class': 'export-config-header ped-header'}).insert("Which of the following should be reflected in the affected column in PED file? ");
                 pedContainer.insert(label.wrap('td').wrap('tr', {'class' : "ped-special-options"}));
 
-                hasDisorders && this._addPedOption("disorders", disorders, pedContainer);
-                hasPhenotypes && this._addPedOption("phenotypes", hpos, pedContainer);
-                hasCancers && this._addPedOption("cancers", cancers, pedContainer);
-                hasCandidateGenes && this._addPedOption("candidateGenes", candidateGenes, pedContainer, "candidate genes");
-                hasCausalGenes && this._addPedOption("causalGenes", causalGenes, pedContainer, "confirmed causal genes");
+                var traitsContainner = new Element('table');
+
+                hasDisorders && this._addPedOption("disorders", disorders, traitsContainner);
+                hasPhenotypes && this._addPedOption("phenotypes", hpos, traitsContainner);
+                hasCancers && this._addPedOption("cancers", cancers, traitsContainner);
+                hasCandidateGenes && this._addPedOption("candidateGenes", candidateGenes, traitsContainner, "candidate genes");
+                hasCausalGenes && this._addPedOption("causalGenes", causalGenes, traitsContainner, "confirmed causal genes");
+
+                pedContainer.insert(traitsContainner.wrap('div', {'class': 'scrollable-container', 'style': 'max-height:'+Math.floor(window.innerHeight/2)+'px;'})
```

Comment:
Appeared there was a bug in print dialog resizing function, which is fixed in second commit.